src/MCPClient/lib/clientScripts/copySubmissionDocs.sh

```diff
@@ -8,3 +8,4 @@
 subdir="$1/submissionDocumentation"
 mkdir -p $subdir
 cp -R "$1/$2/data/objects/submissionDocumentation" $subdir
```

Comment:
Not a bash expert, but ```exit 0``` seems clearer that it's always returning 0 (and silencing the error from cp), rather than the ```|| true```.